### DISCUSSION TITLE
Discord bot

### DISCUSSION THREAD

**Discussion by Preguiii** [discordjs/discord.js#8622] [Q&A] [answered]
I'm creating a bot for discord in javascript. I made a command for when a user sends "-ping" the bot informs the user's ping in the chat, but the bot is not responding to the command. Can you help me?

```
if(comando === "ping") {
        const m = await message.channel.send("Ping?");
        m.edit(`Pong! A Latência é ${m.createdTimestamp - message.createdTimestamp}ms. A Latência da API é ${Math.round(client.ping)}ms`);
    }
```

**Comment by almostSouji** [comment, marked as answer]
You will need to send and enable the `Guilds`, `GuildMessages` and `MessageContent` intents in order to receive the `"messageCreate"` event.
If you have that, check if the event even emits. Otherwise, put logs or debug breakpoints throughout your code to find out where execution stops and inspect the environment there (log or inspect relevant conditions and variables).

More on intents: <https://discordjs.guide/popular-topics/intents.html>

**Comment by Sayrix** [comment]
I recommand you to use slash commands instead
